### PR TITLE
BZ#1464036-added information about getting a pv with blank storage class

### PR DESCRIPTION
**Pull request by gaurav-nelson**
https://bugzilla.redhat.com/show_bug.cgi?id=1464036

### CODE REVIEW

**Comment by gaurav-nelson** [comment]
@wongma7 Can you please check this?

**Comment by wongma7** [comment]
lgtm

**Comment by gaurav-nelson** [comment]
@bfallonf PTAL

File: architecture/additional_concepts/storage.adoc [hunk 1]
```diff
@@ -496,11 +496,15 @@ spec:
...
-are configured by the cluster administrator to service one or more storage
-classes. They create a PV on demand that matches the specifications in the PVC,
-if they are able. It is also possible for the cluster administrator to set a
-default `StorageClass` for all PVCs.
+Claims can optionally request a specific `StorageClass` by specifying its name
+in the `storageClassName` attribute. Only PVs of the requested class, ones with the same `storageClassName` as the PVC, can be bound to the PVC. Dynamic provisioners are configured by the
+cluster administrator to service one or more storage classes. They create a PV
+on demand that matches the specifications in the PVC, if they are able.
+
+Cluster administrator can also set a default `StorageClass` for all PVCs. When a
```
Comment (bfallonf) [inline architecture/additional_concepts/storage.adoc]: I'd suggest "The cluster administrator..."

File: architecture/additional_concepts/storage.adoc [hunk 2]
```diff
@@ -496,11 +496,15 @@ spec:
...
 === Storage Class
 
-Claims can optionally request a specific `StorageClass`. Dynamic provisioners
-are configured by the cluster administrator to service one or more storage
-classes. They create a PV on demand that matches the specifications in the PVC,
-if they are able. It is also possible for the cluster administrator to set a
-default `StorageClass` for all PVCs.
+Claims can optionally request a specific `StorageClass` by specifying its name
+in the `storageClassName` attribute. Only PVs of the requested class, ones with the same `storageClassName` as the PVC, can be bound to the PVC. Dynamic provisioners are configured by the
+cluster administrator to service one or more storage classes. They create a PV
```
Comment (bfallonf) [inline architecture/additional_concepts/storage.adoc]: Active: "The cluster administrator can configure dynamic provisioners to service one or more storage classes."

**Comment by bfallonf** [comment]
@gaurav-nelson A couple nits from me. LGTM otherwise.

**Comment by gaurav-nelson** [comment]
Thanks @bfallonf, updated the commit to include your suggestions.

**Comment by gaurav-nelson** [comment]
minor changes, no revision history.

**Pull request merged**